packages/x-charts/src/internals/plugins/corePlugins/useChartInteractionListener/useChartInteractionListener.types.ts

```diff
@@ -1,43 +1,49 @@
 import {
   MoveEvent,
   PanEvent,
+  PanGestureOptions,
   PinchEvent,
+  PinchGestureOptions,
   PressEvent,
   TapEvent,
   TurnWheelEvent,
+  TurnWheelGestureOptions,
 } from '@mui/x-internal-gestures/core';
 import { ChartPluginSignature } from '../../models';
 
 export type ChartInteraction =
   | 'pan'
   | 'panStart'
   | 'panEnd'
-  | 'pinch'
-  | 'pinchStart'
-  | 'pinchEnd'
+  | 'zoomPan'
+  | 'zoomPanStart'
+  | 'zoomPanEnd'
+  | 'zoomPinch'
+  | 'zoomPinchStart'
+  | 'zoomPinchEnd'
   | 'move'
   | 'moveStart'
   | 'moveEnd'
   | 'quickPress'
   | 'quickPressEnd'
-  | 'turnWheel'
+  | 'zoomTurnWheel'
   | 'tap';
 
 export type InteractionListenerResult = { cleanup: () => void };
 
 export type AddInteractionListener = {
   <CustomData extends Record<string, unknown> = Record<string, unknown>>(
-    interaction: 'pan' | 'panStart' | 'panEnd',
+    interaction: 'pan' | 'panStart' | 'panEnd' | 'zoomPan' | 'zoomPanStart' | 'zoomPanEnd',
```

Comment:
I don't get what are doing those new interactions. What's the difference between a `pan` and a `zoomPan`?